test/parallel/xyk-pallet.multiswap2hops.test.ts

```diff
@@ -0,0 +1,209 @@
+/*
+ *
+ * @group multiswap
+ */
+import { getApi, initApi } from "../../utils/api";
+import {
+  multiSwapBuy,
+  multiSwapSell,
+  calculate_sell_price_id_rpc,
+} from "../../utils/tx";
+import { ExtrinsicResult } from "../../utils/eventListeners";
+import { BN } from "@polkadot/util";
+import { User, AssetWallet } from "../../utils/User";
+import { getUserBalanceOfToken } from "../../utils/utils";
+import { setupApi, setup5PoolsChained } from "../../utils/setup";
+import {
+  getBalanceOfPool,
+  getEventResultFromMangataTx,
+} from "../../utils/txHandler";
+import { BN_ONE, BN_TEN_THOUSAND, BN_ZERO } from "@mangata-finance/sdk";
+import {
+  EVENT_METHOD_PAYMENT,
+  EVENT_SECTION_PAYMENT,
+  MGA_ASSET_ID,
+} from "../../utils/Constants";
+
+jest.spyOn(console, "log").mockImplementation(jest.fn());
+jest.setTimeout(1500000);
+
+const successMultiSwapBuyEventName = "AssetsSwapped";
+const successMultiSwapSellEventName = "AssetsSwapped";
```

Comment:
Should we use 2 names here if they have an equal result?

Reply:
right ! haha

Reply:
fixed!